requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+nomkl
```

Comment:
To keep pip compatibility let's put this inside `Travis-CI` like [this](https://github.com/sgrid/pysgrid/pull/87/files).